numcodecs/tests/test_blosc.py

```diff
@@ -47,6 +47,10 @@
     np.random.randint(0, 2**60, size=1000, dtype='u8').view('m8[ns]'),
     np.random.randint(0, 2**25, size=1000, dtype='u8').view('M8[m]'),
     np.random.randint(0, 2**25, size=1000, dtype='u8').view('m8[m]'),
+    (-np.random.randint(-2**30, 2**30, size=1000, dtype='i8')).view('M8[ns]'),
```

Comment:
Out of interest, if this is random numbers between -2**30 and 2**30, why do we need to take the negative of the array? Is it not enough to specify the limits within dancing?